### PR TITLE
[modular][ready]Prison rainbow, more prison uniforms for the erp mains

### PR DESCRIPTION
**Pull request by thestubborn**
## About The Pull Request

prison stuff good, see my last loadout pr/old base for the outfits

## Why It's Good For The Game

prison man good

## Changelog
:cl: Charlie Bronson
add: New outfits for prisoners
add: Robe for chaplains

/:cl:

### CODE REVIEW

File: modular_skyrat/modules/customization/modules/clothing/under/security.dm [hunk 1]
```diff
@@ -48,3 +48,94 @@
 	heat_protection = CHEST|ARMS
 
 // SEC GENERAL
+
+// PRISONER
+/obj/item/clothing/under/rank/protcustpskirt
+	name = "protective custody prisoner jumpsuit"
```
Comment (Gandalf2k15) [inline modular_skyrat/modules/customization/modules/clothing/under/security.dm:54]: You could just make all these a subtype of the first

Comment (thestubborn) [reply modular_skyrat/modules/customization/modules/clothing/under/security.dm:54]: what would that do?

Comment (Gandalf2k15) [reply modular_skyrat/modules/customization/modules/clothing/under/security.dm:54]: Means you don't have to copy pasta code and it's easier to find and spawn, better code practice

Comment (thestubborn) [reply modular_skyrat/modules/customization/modules/clothing/under/security.dm:54]: ahh, on it

Comment (thestubborn) [reply modular_skyrat/modules/customization/modules/clothing/under/security.dm:54]: sorted

**Comment by Gandalf2k15** [comment]
Aaaah so with subtypes it works like this:
/obj/item/thing
Name = "thing"
Icon state = "default"
Icon ="/icon/Icon.dmi"
Armor = armor

/obj/item/thing/green
Name = green thing
Icon state = green

See how you only put in the things that are different?

**Comment by Gandalf2k15** [comment]
Ack I'll do it later

**Comment by heretomakecomments** [comment]
These don't seem to actually show up in game.

**Comment by thestubborn** [comment]
> 
> 
> These don't seem to actually show up in game.

they're under job related in the loadout

**Pull request merged**